### ISSUE TITLE
You need at least Xcode 7.0 when you are on OS X 10.11 El Capitan (you have version 10.0)

### ISSUE THREAD

**Issue by codeWonderland** [ios-control/ios-deploy#348]
# Expected behavior
I was attempting to install ios-deploy, but the installer thinks my system is out of date. 
The command I used to install ios-deploy was: 
`sudo npm i -g ios-deploy --unsafe-perm --allow-root`

# Actual behavior.
The install failed with the following error: 
`You need at least Xcode 7.0 when you are on OS X 10.11 El Capitan (you have version 10.0)`

# Steps to reproduce the problem
Attempt an install on a Mac with Xcode 10.0

# System Specs
MacBook Pro running macOS Mojave

Please run the commands below in your Terminal.app and include it in the issue. Check when done and include results below.

- [x] 1. system_profiler SPSoftwareDataType
```
System Software Overview:
      System Version: macOS 10.14 (18A314h)
      Kernel Version: Darwin 18.0.0
      Boot Volume: Macintosh HD
      Boot Mode: Normal
      Computer Name: Alice_Easter
      User Name: Alice Easter (easter)
      Secure Virtual Memory: Enabled
      System Integrity Protection: Disabled
      Time since boot: 2 days 1:46
```
- [x] 2. ios-deploy -V
ios-deploy not installed
- [x] 3. xcodebuild -version
```
Xcode 10.0
Build version 10L176w
```
- [x] 4. xcode-select --print-path
`/Applications/Xcode-beta.app/Contents/Developer`
- [x] 5. gcc --version
```
Configured with: --prefix=/Applications/Xcode-beta.app/Contents/Developer/usr --with-gxx-include-dir=/Applications/Xcode-beta.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk/usr/include/c++/4.2.1
Apple LLVM version 10.0.0 (clang-1000.10.25.5)
Target: x86_64-apple-darwin18.0.0
Thread model: posix
InstalledDir: /Applications/Xcode-beta.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```
- [x] 6. lldb --version
```
lldb-1000.0.18.3
  Swift-4.2
```

**Comment by ndrake** [comment]
A PR was merged to fix this: https://github.com/ios-control/ios-deploy/pull/342 but a new release of ios-deploy hasn't been created yet.

**Comment by theSMguy** [comment]
I also tried this, clearing npm cache did not work (I'm working with react-native)

**Comment by Oyalunasamuel** [comment]
has anyone been able to come up with a solution to this I am stuck here

**Comment by shazron** [comment]
Should be fixed with the upcoming 1.9.3 releaes

**Comment by shazron** [comment]
1.9.3 published with this fix: https://github.com/ios-control/ios-deploy/releases/tag/1.9.3
